src/vm/eventpipeconfiguration.cpp

```diff
@@ -75,7 +80,7 @@ void EventPipeConfiguration::Initialize()
     CONTRACTL_END;
 
     // Create the configuration provider.
-    m_pConfigProvider = CreateProvider(SL(s_configurationProviderName), NULL, NULL);
+    m_pConfigProvider = new EventPipeProvider(this, SL(s_configurationProviderName), NULL, NULL);
```

Comment:
Why not go through CreateProvider?  Are you trying to avoid calling Register on it?

Reply:
Yes, the configuration provider should not be registered because it will be managed separately. 

That is what I inferred to be the correct behavior based on the way it was originally created before adding the CreateProvider function

Reply:
I'm not aware of anything special about it.  Let's treat it as a normal provider unless we have a reason not to.